app/frontend/build/webpack.dev.conf.js

```diff
@@ -93,7 +97,13 @@ const devWebpackConfig = merge(baseWebpackConfig, {
     // in a separate chunk, similar to the vendor chunk
     // see: https://webpack.js.org/plugins/commons-chunk-plugin/#extra-async-commons-chunk
     new webpack.optimize.CommonsChunkPlugin({
-      names: ['header', 'registry', 'documents', 'footer'],
+      names: [
+        'header',
+        'registry',
+        'documents',
+        'footer',
+        'submissions'
```

Comment:
The order that these are specified in is actually important. Please move submissions above footer.